app/vmui/packages/vmui/src/utils/time.ts

```diff
@@ -250,3 +250,33 @@ export const getBrowserTimezone = () => {
     region: isValid ? timezone : "UTC",
   };
 };
+
+export const getNanoTimestamp = (dateStr: string): bigint => {
```

Comment:
Any chance you can add tests to verify correctness of this function?

Reply:
Added tests